test/keycloak-connect-rest-enforcer-spec.js

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2016 Red Hat Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+'use strict';
+
+const admin = require('./utils/realm');
+const NodeApp = require('./fixtures/node-console/index').NodeApp;
+
+const test = require('blue-tape');
+const roi = require('roi');
+const getToken = require('./utils/token');
+
+const realmName = 'service-node-realm';
```

Comment:
@pedroigor I tried to test it with Keycloak server from master. But I believe that's not the reason why tests are failing.

There's a duplicate with https://github.com/keycloak/keycloak-nodejs-connect/blob/master/test/keycloak-connect-rest-spec.js#L26. That's an easy fix, just rename to something like `policy-enforcer-realm`.

Even after renaming it, I get 3 tests failing. You can reproduce it by doing the following steps:

1. clone/build Keycloak from master
2. Start the server
```
#!/bin/bash

set -v

# Start the server
$KEYCLOAK/bin/add-user-keycloak.sh -u admin -p admin

# Start the server
$KEYCLOAK/bin/standalone.sh -Djava.net.preferIPv4Stack=true \
                            -Dkeycloak.migration.action=import \
                            -Dkeycloak.migration.provider=singleFile \
                            -Dkeycloak.migration.file=./test/fixtures/auth-utils/nodejs-test-realm.json \
                            -Dkeycloak.migration.strategy=OVERWRITE_EXISTING > keycloak.log 2>&1 &
``` 
3. Run `make tests`

Reply:
Let me try it out. Thanks.

Reply:
I'm able now to get `make tests` running. But I had to change `keycloak-connect-rest-enforcer-spec.js` to also pass the `realmName` when calling `getToken({ realmName })`.